load_balancer/test/integration/load_balancerTest.js

```diff
@@ -0,0 +1,243 @@
+console.log('======integration tests on load balancer======\n');
+
+const chai = require('chai');
+const dirtyChai = require('dirty-chai');
+const request = require('request');
+const nock = require('nock');
+const dns = require('dns');
+const sinon = require('sinon');
+const { check } = require('../../../util/environmentCheck.js');
+const testData = require('./data/submission.json');
+// eslint-disable-next-line import/no-dynamic-require
+const configData = require(`../../${process.env.LBCONFIG}`);
+const lbUrl = `https://${configData.load_balancer.hostname}:${configData.load_balancer.port}`;
+const sandbox = sinon.createSandbox();
+
+let loadBalancer;
+let logStub;
+
+check('LBCONFIG');
+
+chai.should();
+chai.use(dirtyChai);
+
+const activateNocks = function activateNocks() {
+  if (!nock.isActive()) {
+    nock.activate();
+  }
+};
+
+const cleanNocks = function cleanNocks() {
+  nock.cleanAll();
+  nock.restore();
+};
+
+const stubConsole = function stubConsole() {
+  logStub = sandbox.stub(console, 'log');
+};
+
+const restoreConsole = function restoreConsole() {
+  logStub = {};
+  sandbox.restore();
+};
+
+const startLoadBalancer = function startLoadBalancer() {
+  delete require.cache[require.resolve('../../load_balancer.js')];
+  // eslint-disable-next-line global-require
+  loadBalancer = require('../../load_balancer.js');
+};
+
+const connectionCheckNock = function connectionCheckNock(url, returnVal) {
```

Comment:
This function name can be better. Its job seems to be setting up a mock status return value for an execution node. The name should reflect that intent.

Reply:
Sir, how about mockNodeStatus?

Reply:
may be, ´mockENStatus`?

Reply:
Alright sir, I'll do that.